corehq/apps/export/templates/export/new_customize_export.html

```diff
@@ -0,0 +1,309 @@
+{% extends "settings/bootstrap2/base_template.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block js %}
+    {{ block.super }}
+    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
+    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
+    <script src="{% static 'jquery-ui-1.8.23-legacy/jquery-ui.min.js' %}"></script>
+    <script src="{% static 'export/js/models.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+    <script>
+        $(function () {
+            var translations = {
```

Comment:
No but this file is slowly going to get refactored. This is all stuff from
the old customize_export.html. I have another branch where I'm working on
stuff like this

On Wednesday, February 3, 2016, Simon Kelly <notifications@github.com>
wrote:

> In corehq/apps/export/templates/export/new_customize_export.html
> <https://github.com/dimagi/commcare-hq/pull/10157#discussion_r51696700>:
>
> > +{% extends "settings/bootstrap2/base_template.html" %}
> > +{% load hq_shared_tags %}
> > +{% load i18n %}
> > +
> > +{% block js %}
> > +    {{ block.super }}
> > +    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
> > +    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
> > +    <script src="{% static 'jquery-ui-1.8.23-legacy/jquery-ui.min.js' %}"></script>
> > +    <script src="{% static 'export/js/models.js' %}"></script>
> > +{% endblock %}
> > +
> > +{% block js-inline %}{{ block.super }}
> > +    <script>
> > +        $(function () {
> > +            var translations = {
>
> Is this used somewhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10157/files#r51696700>.
>